packages/vue-components/src/panels/PanelBase.js

```diff
@@ -141,6 +141,9 @@ export default {
           Thus, we need to reset the maxHeight to its current height for collapse transition to work.
         */
         this.$refs.panel.style.maxHeight = `${this.$refs.panel.scrollHeight}px`;
+        if (this.$refs.panel.scrollHeight > 300) {
+          this.$refs.panel.style.transition = 'max-height 0s ease-in-out';
```

Comment:
On that note, I noticed the jQuery scroll animation isn't disabled too, was thinking it should (sorry that the issue description wasn't clear enough!)

Reply:
@tlylt @eeliana in case you missed this.

It's nowhere near as disorienting with the panel height transition removed, but I'm not entirely sure if this should be removed as well. (personally, still favouring removal as there is a slight time lag till the scroll takes place)

If it looks ok / better to you guys feel free to keep it :)